src/main/java/inf112/skeleton/app/Game/MyGame.java

```diff
@@ -84,11 +85,12 @@ public MyGame() {
 
     @Override
     public void create() {
-        map = new Map("map_v1.tmx");
+        map = new Map(MapRenderer.whatMapToCreateString());
```

Comment:
Denne linjen ser ut for å gi en hel haug med feilmeldinger når man kjører tester.

Reply:
Det kan ha noe med static/non-static referering?